pom.xml

```diff
@@ -16,19 +16,19 @@
   -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+    <modelVersion>4.0.0</modelVersion>
     <groupId>org.axonframework.extensions.kafka</groupId>
     <artifactId>axon-kafka-parent</artifactId>
     <version>4.6.0-SNAPSHOT</version>
+
+    <packaging>pom</packaging>
+
     <modules>
         <module>kafka</module>
         <module>kafka-spring-boot-autoconfigure</module>
         <module>kafka-spring-boot-starter</module>
-        <module>kafka-axon-example</module>
+        <!-- Example module is inside the profile and can be switched off -->
```

Comment:
I think it is good to build the example by default. It would allow to start this example and run it... So on-by-default is good... 

I just see no value in publishing a Maven POM, Maven JAR, Maven Sources JAR, JavaDoc JAR and GPG signatures of all those files to OSS Maven central, since I bleieve than nobody would ever need to consume this.

So for usual build: `./mvnw clean install`, to speed it up: `./mvnw clean install -DskipExamples`, for build during deployment: `./mvnw clean deploy -DskipTests -DskipExamples`

Reply:
Ow, then I didn't check the changes correctly. I assumed as you've removed it here that it's no longer a part of a regular build.
From your response, I gather that's a wrong assumption on my part, correct?

Note that I do agree. The sample definitely doesn't have to go to Maven Central.

Reply:
Yes. It is removed from the "regular" modules list, but is still inside the modules list, which is combined from the regular + profiles, since the "example" profile by default. And because it is in profile, you can deactivate it ...

Reply:
Gotcha. Because the `skipExamples` isn't defined for a regular build, it's false and hence the sample is included. The workflow in turn depends on this for a normal build, but when going to Sonatype you've set `skipExamples` to true.